pkg/services/ngalert/metrics/multi_org_alertmanager.go

```diff
@@ -28,5 +37,112 @@ func NewMultiOrgAlertmanagerMetrics(r prometheus.Registerer) *MultiOrgAlertmanag
 			Name:      "active_configurations",
 			Help:      "The number of active Alertmanager configurations.",
 		}),
+		aggregatedMetrics: NewAlertmanagerAggregatedMetrics(registries),
+	}
+
+	// These metrics use a different registration method as the struct itself represents a custom collector.
+	// There's no way to "auto-register" a collector.
+	r.MustRegister(moa.aggregatedMetrics)
+
+	return moa
+}
+
+// RemoveOrgRegistry removes the *prometheus.Registry for the specified org. It is safe to call concurrently.
+func (moa *MultiOrgAlertmanager) RemoveOrgRegistry(id int64) {
+	moa.registries.RemoveUserRegistry(strconv.FormatInt(id, 10), false)
+}
+
+// GetOrCreateOrgRegistry gets or creates a *prometheus.Registry for the specified org. It is safe to call concurrently.
+func (moa *MultiOrgAlertmanager) GetOrCreateOrgRegistry(id int64) prometheus.Registerer {
+	sid := strconv.FormatInt(id, 10)
+	reg := moa.registries.GetRegistryForUser(sid)
+	if reg != nil {
+		return reg
+	}
+
+	result := prometheus.NewRegistry()
+	moa.registries.AddUserRegistry(sid, result)
+
+	return result
+}
+
+// AlertmanagerAggregatedMetrics are metrics collected directly from the registry.
```

Comment:
I've created a separate struct as I have close to 30 other metrics that I will include next.

It felt a bit confusing to have `metrics.Alertmanager` and another struct that that reads "some form of alertmanager metrics" but I ended up settling for `AlertmanagerAggregatedMetrics` as this is exactly what it is doing.